packages/block-library/src/list/v2/save.js

```diff
@@ -1,5 +1,12 @@
-function Save() {
-	return <div>List block v2</div>;
-}
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
-export default Save;
+export default function save() {
+	return (
+		<ul { ...useBlockProps.save() }>
+			<InnerBlocks.Content />
```

Comment:
Could we add some checks to return null if there are no inner blocks? It is very easy to create an empty list item for the user is very hard to remove. Or maybe we can automatically remove the list block if it becomes empty?

Reply:
we should add a shortcut later that removes it on backspace